File: square/src/main/java/com/raxdenstudios/square/utils/FragmentUtils.java
package com.raxdenstudios.square.utils;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentTransaction;

/**
 * Created by Ángel Gómez on 10/06/2017.
 */

public class FragmentUtils {

    public static void loadFragment(FragmentManager fm, int containerId, Fragment fragment) {
        FragmentTransaction transaction = fm.beginTransaction();
        String tag = fragment.getClass().getSimpleName();
        transaction = transaction.replace(containerId, fragment, tag);
        transaction.commit();
    }

    public static Fragment getFragment(FragmentManager fm, int containerId) {
        Fragment fragment = null;
        if (fm != null) {
            fragment = fm.findFragmentById(containerId);
        }
        return fragment;
    }

}

File: square/src/main/java/com/raxdenstudios/square/utils/FragmentUtils.kt
package com.raxdenstudios.square.utils

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager

/**
 * Created by Ángel Gómez on 10/06/2017.
 */

object FragmentUtils {

    fun <TFragment : Fragment> loadFragment(fragmentManager: FragmentManager, containerId: Int, fragment: TFragment) {
        fragmentManager
                .beginTransaction()
                .replace(containerId, fragment, fragment.javaClass.simpleName)
                .commit()
    }

    fun <TFragment : Fragment> getFragment(fragmentManager: FragmentManager, containerId: Int): TFragment? =
            fragmentManager.findFragmentById(containerId) as TFragment

}
